sql/mysql_db_structure.sql

```diff
@@ -68,7 +72,7 @@ CREATE TABLE `clarification` (
 CREATE TABLE `configuration` (
   `configid` int(4) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Unique ID',
   `name` varchar(32) NOT NULL COMMENT 'Name of the configuration variable',
-  `value` longtext NOT NULL COMMENT 'Content of the configuration variable (JSON encoded)',
+  `value` longtext NOT NULL COMMENT 'Content of the configuration variable (JSON encoded)(DC2Type:json)',
```

Comment:
The `DC2Type` thing is needed by Doctrine when one uses a non-built-in type, so it knows what to do with the column.